views/jira-select-github-cloud-app.hbs

```diff
@@ -44,7 +44,7 @@
             subHeader=(concat "We found the following GitHub apps for your server, " "<b>" serverUrl "</b>")
         }}
         <!-- servers refer to the array of GHE apps/servers-->
-        {{> select-server-table
+        {{> select-table
```

Comment:
** No need to action on this in this PR **
This kind of bug is annoying, not sure what's better way we can test it.

Feel free to close this thread.